TopAnalysis/python/postprocessing/fcncTriLepton.py

```diff
@@ -116,8 +117,16 @@ def analyze(self, event):
 fcnc_ElElEl_2016 = lambda : FCNCTriLepton(mode="ElElEl", eleId="cutBased_Sum16")
 fcnc_ElMuMu_2016 = lambda : FCNCTriLepton(mode="ElMuMu", eleId="cutBased_Sum16")
 fcnc_MuElEl_2016 = lambda : FCNCTriLepton(mode="MuElEl", eleId="cutBased_Sum16")
+fcnc_NPLMuMuMu_2016 = lambda : FCNCTriLepton(mode="NPLMuMuMu", eleId="cutBased_Sum16")
```

Comment:
(related with the previous comment - say we have one more argument such as eleVidMap="vid.....Sum16"

Reply:
BTW, better to comment out at this moment as we decided not to produce NPL's at this moment.